### PR TITLE
docs(influxdb_logs/metrics sink): autogen docs

### LINKED ISSUE #16021: Convert the `influxdb_logs` sink to use auto-generated docs

### LINKED ISSUE #16022: Convert the `influxdb_metrics` sink to use auto-generated docs

### PR DESCRIPTION
**Pull request by davidhuie-dd**
Closes #16021, closes #16022

### CODE REVIEW

File: src/sinks/influxdb/logs.rs [hunk 1]
```diff
@@ -46,16 +46,23 @@ pub struct InfluxDbLogsConfig {
     ///
     /// This field is deprecated, and `measurement` should be used instead.
     #[configurable(deprecated)]
```
Comment (spencergilbert) [inline src/sinks/influxdb/logs.rs:48]:
```suggestion
    #[configurable(deprecated = "This field is deprecated, and `measurement` should be used instead.")]
```

**Review by spencergilbert — CHANGES_REQUESTED** [review]
There were a couple of examples dropped (ex `token` `username`) but I'd approve it without those added since they're so simple.

**Review by spencergilbert — APPROVED** [review]

File: src/sinks/influxdb/logs.rs (hunk 1 above)
Comment (spencergilbert) [reply src/sinks/influxdb/logs.rs]: I betrayed you! Didn't format it properly 😭

File: src/sinks/influxdb/logs.rs [hunk 2]
```diff
@@ -44,18 +44,24 @@ pub struct InfluxDbLogsConfig {
...
     pub namespace: Option<String>,
 
     /// The name of the InfluxDB measurement that will be written to.
+    #[configurable(metadata(docs::examples = "vector-logs"))]
     pub measurement: Option<String>,
 
     /// The endpoint to send data to.
+    ///
+    /// This should be a full HTTP URI, including the scheme, host, and port.
+    #[configurable(metadata(docs::examples = "http://localhost:8086/"))]
```
Comment (neuronull) [inline src/sinks/influxdb/logs.rs:58]: nit (I know it was this way in the existing docs)

```suggestion
    #[configurable(metadata(docs::examples = "http://localhost:8086"))]
```

**Review by neuronull — APPROVED** [review]

**Pull request merged**